etc/gather_coverage.jl

```diff
@@ -6,8 +6,9 @@ using Coverage
 import JSON
 
 # gather coverage
-data = Codecov.process_folder(pwd())
-data_dict = Codecov.to_json(data)
+data = process_folder("src")
+data = append!(data, process_folder("pkg"))
```

Comment:
Hrm, I must be missing something -- for some reason, this collects data in `pkg`, but fails to collect it in `src`...

Reply:
I am not sure what you mean. In this branch, `src` only has `GAP.jl`, which gets collected. Also, the `.c` files get collected. So I guess it is fine.

Reply:
Err, no, `src/GAP.jl` does *not* get collected, as the codecov report clearly showed, and as now also https://codecov.io/gh/oscar-system/GAP.jl shows. So merging this unfortunately was incorrect. I'll revert it.